Place4AllBackend/src/Common/Place4AllBackend.Infrastructure/Seeders/UserSeeder.cs

```diff
@@ -60,11 +62,11 @@ public class UserSeeder
                 Province = "Zaragoza"
             }
         };
-        var userLaura = new ApplicationUser 
+        var userLaura = new ApplicationUser
         {
             Name = "Laura",
             LastName = "Gil",
-            Age = 27,
+            BirthDate = new DateTime(1996, 02, 15),
```

Comment:
ayyy mi cumple! jajaja Entonces ahora la edad a la hora de registrarse is required

Reply:
La edad ya no hace falta ya que será una fecha de nacimiento

Reply:
Faltaría mi fecha de cumpleaños, espero que no dé problemas el usuario, al iniciar sesión en un futuro